site/content/en/docs/Installation/_index.md

```diff
@@ -45,17 +43,17 @@ Each version of Agones supports a specific version of Kubernetes. When a new ver
 
 The following table lists recent Agones versions and their corresponding required Kubernetes versions:
 
-| Agones version | Kubernetes version |
-| -------------- | ------------------ |
-| 1.29           | 1.24               |
-| 1.28           | 1.23               |
-| 1.27           | 1.23               |
-| 1.26           | 1.23               |
-| 1.25           | 1.22               |
-| 1.24           | 1.22               |
-| 1.23           | 1.22               |
-| 1.22           | 1.21               |
-| 1.21           | 1.21               |
+| Agones version | Kubernetes version   |
```

Comment:
Starting with the next release we should change the column header to Kubernetes versions (plural).